README.md

```diff
@@ -1 +1,158 @@
-# Final-capstone-Frontend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] You've included frontend tests in your project, but 4 tests are failing:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/39b85c4f-b04b-473e-af3b-1e3bbbd2a0bf)

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/8f3cdaf5-ef4d-47e5-916f-a447a39cc0c7)


Kindly, fix those tests so your app's integrity is secured. 👍